pywinauto/win32structures.py

```diff
@@ -131,11 +131,13 @@ def _reduce(self):
 UINT = c_uint
 WCHAR = c_wchar
 WORD = c_ushort
+LRESULT = wintypes.LPARAM

```

Comment:
See wintypes definitions of WPARAM/LPARAM below. It's, probably, more accurate than ours but it may destabilize the code. We'll do this change a bit later. Now we'll stick to wintypes.LPARAM for LRESULT as it is already in use by win32_hooks.
```python
# WPARAM is defined as UINT_PTR (unsigned type)
# LPARAM is defined as LONG_PTR (signed type)
if sizeof(c_long) == sizeof(c_void_p):
    WPARAM = c_ulong
    LPARAM = c_long
elif sizeof(c_longlong) == sizeof(c_void_p):
    WPARAM = c_ulonglong
    LPARAM = c_longlong
```

Reply:
Sure, let's do it in 0.7.0. Filed issue #650.